distribution/lib/Standard/Base/0.0.0-dev/src/Data/Time/Date.enso

```diff
@@ -446,17 +446,13 @@ type Date
              import Standard.Base.Data.Time.Date_Period
 
              example_add = Date.new 2020 + Date_Period.Month
-    + : Period -> Date ! (Time_Error | Illegal_Argument)
-    + self amount =
+    + : Period | Date_Period -> Date ! (Time_Error | Illegal_Argument)
+    + self amount:(Period|Date_Period) =
```

Comment:
Personally I prefer
```suggestion
    + self (amount : Period|Date_Period) =
```
as IMO it looks nicer. But that is very subjective, feel free to ignore, just sharing my preferences :)